source/features/repo-header-info.tsx

```diff
@@ -55,7 +55,7 @@ async function add(repoLink: HTMLAnchorElement): Promise<void> {
 }
 
 function init(signal: AbortSignal): void {
-	observe('header .AppHeader-context-full li:last-child a', add, {signal});
+	observe('header .AppHeader-context-full li:last-child a:has(span.AppHeader-context-item-label)', add, {signal});
```

Comment:
Do safari still not supports `:has()` selector? If it is, we can use below

```suggestion
	observe('header .AppHeader-context-full li:last-child > a', add, {signal});
```

Reply:
Why not like this?

```suggestion
	observe('header .AppHeader-context-full li:last-child a.AppHeader-context-item', add, {signal});
```

Firefox does not support `:has` yet by default, see #5797. Note that the `> a` selector would probably break #6926.

Reply:
@FloEdelmann Thank you! I missed that..